documentation/publishing.md

```diff
@@ -1,8 +1,9 @@
 # Publishing
 
-We use Lerna to create changelogs and releases for all our packages and to publish them to npm.
+We use a Release Please GitHub action to create changelog and release PRs for all our packages.
```

Comment:
```suggestion
We use a [Release Please](https://github.com/googleapis/release-please) GitHub action to create a changelog and release PRs for all our packages.
```

Reply:
Good idea, but I'll link directly to the action itself